sdk/netapp/Azure.ResourceManager.NetApp/api/Azure.ResourceManager.NetApp.netstandard2.0.cs

```diff
@@ -1540,4 +1596,10 @@ public partial class SnapshotPolicyWeeklySchedule
         public int? SnapshotsToKeep { get { throw null; } set { } }
         public long? UsedBytes { get { throw null; } set { } }
     }
+    public partial class VolumeRelocationProperties
```

Comment:
Since in this RP, Volume corresponds to `NetappVolume`, could we rename this accordingly?
```suggestion
    public partial class NetappVolumeRelocationProperties
```